client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -280,61 +280,29 @@ export class DisplayTable extends React.Component {
       .reverse()
       .value();
 
-    const {
-      paginate_by,
-      paginated_data,
-      change_paginate_by,
-      show_pagination_controls,
-      page_selector,
-    } = (() => {
-      const paginate_by =
-        size_to_paginate && _.size(sorted_filtered_data) > size_to_paginate
-          ? state_paginate_by
-          : 0;
-
-      const num_pages_exact = paginate_by
-        ? _.size(sorted_filtered_data) / paginate_by
-        : -1;
-
-      const num_pages =
-        num_pages_exact >= 0 && _.isInteger(num_pages_exact) //if we have the exact amount of items to fill an entire page, it will start a new empty page
-          ? num_pages_exact - 1 //thus we want to get rid of the extra page (as it has no content)
-          : _.toInteger(num_pages_exact); //start counting from 0 (-1 from the literal amount of pages)
-
-      const paginated_data = _.chunk(
-        sorted_filtered_data,
-        paginate_by || _.size(sorted_filtered_data)
-      )[current_page];
-
-      const change_page = (page) => {
-        this.setState({ current_page: page });
-        this.first_data_ref.current.focus();
-      };
-
-      const change_paginate_by = (new_paginate_by) => {
-        this.setState({ state_paginate_by: new_paginate_by, current_page: 0 });
-      };
-
-      const show_pagination_controls = num_pages >= 0;
-
-      const page_selector = show_pagination_controls && (
-        <PageSelector
-          num_pages={num_pages}
-          current_page={current_page}
-          change_page={change_page}
-          show_select
-          num_col={_.size(visible_ordered_col_keys)}
-        />
-      );
+    const paginated_data = _.chunk(sorted_filtered_data, page_size);
+
+    const number_of_pages = paginated_data.length;
+    const show_pagination_controls = number_of_pages > 1;
```

Comment:
Small bug I introduced here, can lose the pagination controls if you pick a page size that results in `number_of_pages === 1`. A better idea would probably be to make `show_pagination_controls` independent of state (think I commented this on the base PR as a design point, but I don't think the controls should go away even if a filter is applied that makes them useless). Should probably just be a boolean prop. At most, the "smart" version of the display table to check the initial items against the `page_size` prop and switch `show_pagination_controls` to false if it is unnecessary.